modopt/opt/algorithms/admm.py

```diff
@@ -0,0 +1,343 @@
+"""ADMM Algorithms."""
+import numpy as np
+
+from modopt.base.backend import get_array_module
+from modopt.opt.algorithms.base import SetUp
+from modopt.opt.cost import AbstractcostObj
+
+
+class ADMMcostObj(AbstractcostObj):
+    r"""Cost Object for the ADMM problem class.
+
+    Compute :math:`f(u)+g(v) + \tau \| Au +Bv - b\|^2`
+
+    Parameters
+    ----------
+    cost_funcs: 2-tuples of callable
```

Comment:
In order for the intersphinx mapping to work the argument her has to be valid data type. So just `tuple`, then you can elaborate on the line below.